### PR TITLE
Add metric auto-updates chart

### PR DESCRIPTION
**Pull request by WalshyDev**
**This requires the main PR merged (and preferably an RC out) before this can be merged**

### CODE REVIEW

File: src/main/java/dev/walshy/sfmetrics/charts/MetricAutoUpdatesChart.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package dev.walshy.sfmetrics.charts;
+
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import org.bstats.bukkit.Metrics;
+
+public class MetricAutoUpdatesChart extends Metrics.SimplePie {
```
Comment (TheBusyBiscuit) [inline src/main/java/dev/walshy/sfmetrics/charts/MetricAutoUpdatesChart.java]: Missing s in the class name.

File: src/main/java/dev/walshy/sfmetrics/MetricsModule.java [hunk 2]
```diff
@@ -40,8 +41,9 @@ public static void start() {
         metrics.addCustomChart(new CommandChart());
         metrics.addCustomChart(new ServerSizeChart());
         metrics.addCustomChart(new CompatibilityModeChart());
-        metrics.addCustomChart(new MetricsVersionChart());
 
-        SlimefunPlugin.instance().getLogger().info("Now running MetricsModule v" + VERSION);
+        // Metric specific
+        metrics.addCustomChart(new MetricsVersionChart());
+        metrics.addCustomChart(new MetricAutoUpdatesChart());
```
Comment (TheBusyBiscuit) [inline src/main/java/dev/walshy/sfmetrics/MetricsModule.java]: You forgot to update the class name in this class...

**Comment by TheBusyBiscuit** [comment]
Also why do we have no Java CI here?

File: src/main/java/dev/walshy/sfmetrics/MetricsModule.java (hunk 2 above)
Comment (WalshyDev) [reply src/main/java/dev/walshy/sfmetrics/MetricsModule.java]: Give me a minute! I just did this with nano

**Comment by TheBusyBiscuit** [comment]
Ah okay, the fail is just due to the missing method in Slimefun

**Comment by WalshyDev** [comment]
This is expected to fail currently.

**Comment by TheBusyBiscuit** [comment]
I know

**Comment by TheBusyBiscuit** [comment]
Also, I have added the two new charts on bStats:
https://bstats.org/plugin/bukkit/Slimefun/4574#metrics_version

**Pull request merged**